onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, model, per_channel, reduce_range, mode, static, weight_qType,
         self.quantized_value_map = {}
         # some output from nodes will be quantized, yet itself should be treat as existing so
         # no dequantized will be applied when needed later
-        self.generated_value_names = {}
+        self.generated_value_names = self.model.get_non_initializer_inputs()
```

Comment:
Nice concept, could we also add both initializer  and input here?

Reply:
We have a logic to remove quantized initializer below. If the initializer is used by both quantized op and fp32 op, there will be no logic to dequantize the quantized initializer back.